medusa/show_updater.py

```diff
@@ -74,7 +74,14 @@ def run(self, force=False):
             # Move through each show from the expired season cache table. And run the full show or per season update.
             for show_id in expired_seasons[indexer]:
                 # Loop through the shows.
+
+                # Get the show object and check, to prevent issues further down the line.
                 show = Show.find_by_id(app.showList, indexer, show_id)
+                if not show:
```

Comment:
This is also a check, that might not be needed if the table is updated properly when shows are deleted.